db/migrate/20230119121322_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.text :text
```

Comment:
- [ ] @topeogunleye kindly remove this field as it doesn't exist in the Entity Relationship Diagram provided
![image](https://user-images.githubusercontent.com/61408860/213536198-1bd7b393-90c4-46d9-b6ec-e13bcc740e8d.png)